en/08_Changelogs/6.0.0.md

```diff
@@ -11,14 +11,16 @@ title: 6.0.0 (unreleased)
   - [Changes to default cache adapters](#caching)
   - [Changes to scaffolded form fields](#scaffolded-fields)
   - [`SiteTree` uses form field scaffolding](#sitetree-scaffolding)
-  - [Other new features](#other-new-features)
+  - [Changes to the templating/view layer](#view-layer)
+  - [Other new features and small changes](#other-new-features)
 - [Dependency changes](#dependency-changes)
   - [`intervention/image` has been upgraded from v2 to v3](#intervention-image)
 - [Bug fixes](#bug-fixes)
 - [API changes](#api-changes)
   - [Most extension hook methods are now protected](#hooks-protected)
   - [Strict typing for `Factory` implementations](#factory-strict-typing)
   - [General changes](#api-general)
+- [Full list of removed and changed API (by module, alphabetically)](#api-removed-and-changed)
```

Comment:
This is the name and anchor we gave this section for CMS 5.
Adding now so I can link to it.